charts/tekton-apps/templates/configmap.yaml

```diff
@@ -0,0 +1,43 @@
+{{- range $project := .Values.apps}}
+
+{{ if $project.enabled }}
+{{- range $component := $project.components }}
+{{ $data := dict "project"     $project
+                 "awsConf"     $.Values.aws
+                 "slackConf"   $.Values.slack
```

Comment:
Does the line with `Values.slack` really works on VP? I don't see that it passed in tekton-apps.yaml or values.yaml. Maybe it is redundant?